python/cm/adcm_schema.yaml

```diff
@@ -0,0 +1,617 @@
+---
+# Main config.yaml object should be a list of Objects
+root:
+  match: list
+  item: object
+
+###############################################################################
+#                             O B J E C T S
+###############################################################################
+# There are a number of object types: cluster, provider, host and service
+# All of them has the same struture but with small difference
+
+object:
+  match: dict_key_selection
+  selector: type
+  variants:
+    cluster: cluster_object
+    service: service_object
+    host: host_object
+    provider: provider_object
+    adcm: adcm_object
+
+common_object: &common_object
+  match: dict
+  items: &common_object_items
+    type: string
+    name: string
+    version: version_rule
+    display_name: string
+    description: string
+    edition: string
+    license: string
+    adcm_min_version: version_rule
+    config: config_obj
+    actions: actions_dict
+  required_items: &common_object_required_items
+    - type
+    - name
+    - version
+
+service_object:
+  <<: *common_object
+  items:
+    <<: *common_object_items
+    import: import_dict
+    export: export
+    components: components_dict
+    required: boolean
+    monitoring: monitoring
+
+cluster_object:
+  <<: *common_object
+  items:
+    <<: *common_object_items
+    upgrade: upgrade_list
+    import: import_dict
+    export: export
+
+host_object:
+  <<: *common_object
+  items:
+    <<: *common_object_items
+
+provider_object:
+  <<: *common_object
+  items:
+    <<: *common_object_items
+    upgrade: upgrade_list
+
+adcm_object:
+  <<: *common_object
+  items:
+    <<: *common_object_items
+    upgrade: upgrade_list
+
+export:
+  match: one_of
+  variants:
+    - string
+    - list_of_string
+
+monitoring:
+  match: set
+  variants:
+    - active
+    - passive
+
+components_dict:
+  match: dict
+  default_item: component_item
+
+component_item:
+  match: one_of
+  variants:
+    - none
+    - component_dict
+
+component_dict:
+  match: dict
+  items:
```

Comment:
https://github.com/arenadata/docs_adcm/pull/11/files need to add `bound_to` property

Reply:
also in doc https://docs.arenadata.io/adcm/sdk/config.html#components property `description` is present
where is the fix required?

Reply:
fixed